.github/workflows/nightly_release.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         run: |
           python3 -m pip install pluggy
           python3 -m pip install ruamel.yaml
+          python3 -m pip install numpy
```

Comment:
Should we pin a version for numpy? Or even just install all rasa dependencies? (in case there are more dependencies that might be imported)?

Reply:
I was also thinking just installing all dependencies might be a better fix. I'll make the change, thanks for the suggestion.

Reply:
can we consider the suggestion I wrote in [this ticket](https://rasahq.atlassian.net/browse/ATO-1299)? I think this should be more robust; and we need an alert ;)